admin-dev/themes/new-theme/js/components/components-map.ts

```diff
@@ -102,4 +102,5 @@ export default {
     messageBoxId: 'content-message-box',
     notificationClass: 'contextual-notification',
   },
+  ajaxConfirmation: '#ajax_confirmation',
```

Comment:
Warning, this makes me think about a PR from @Progi1984 which redefined a more "identified" zone for notifications (in a better way IMHO because ajax_confirmation div seemed a bit lost from legacy code)

See this PR and discuss with @Progi1984 maybe https://github.com/PrestaShop/PrestaShop/pull/27347

Reply:
Seen with @atomiix & @NeOMakinG : it's good.